src/hardware/rigol-ds/api.c

```diff
@@ -1034,8 +1030,8 @@ static int dev_acquisition_start(const struct sr_dev_inst *sdi)
 		break;
 	}
 
-	if (devc->data_source == DATA_SOURCE_LIVE && protocol != PROTOCOL_V5)
-		if (rigol_ds_config_set(sdi, ":RUN") != SR_OK)
+	if (devc->data_source == DATA_SOURCE_LIVE)
+		if (rigol_ds_config_set(sdi, ":SING") != SR_OK)
```

Comment:
This is making a lot of sense to me; for a start I was seeing retriggering in :RUN so I think multiple channels were showing inconsistent results; but the other effect here is that you select 'live' grab a frame and look at it quickly, and if you like it then you select 'memory' or 'segment' and this means you *don't* retrigger - you just get the data you previously got in live in a high res fetch.
(Although I'm finding hte timescale there is very screwy in segmented mode)